keyboards/dz60/rules.mk

```diff
@@ -7,7 +7,7 @@ BOOTLOADER = atmel-dfu
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration (just Escape!!)
```

Comment:
The changes to this file should be reverted, and moved to a `rules.mk` fine in your keymap folder